packages/core/src/extensions/Blocks/PreviousBlockTypePlugin.ts

```diff
@@ -93,21 +91,40 @@ export const PreviousBlockTypePlugin = () => {
             const newContentNode = node.node.firstChild;
             if (oldNode && oldContentNode && newContentNode) {
               const newAttrs = {
-                // listType: node.node.attrs.listType,
+                listItemType: newContentNode.attrs.listItemType,
+                listItemIndex: newContentNode.attrs.listItemIndex,
                 headingLevel: newContentNode.attrs.headingLevel,
                 type: newContentNode.type.name,
                 depth: newState.doc.resolve(node.pos).depth,
               };
 
               const oldAttrs = {
-                // listType: oldNode.node.attrs.listType,
+                listItemType: oldContentNode.attrs.listItemType,
+                listItemIndex: oldContentNode.attrs.listItemIndex,
                 headingLevel: oldContentNode.attrs.headingLevel,
                 type: oldContentNode.type.name,
                 depth: oldState.doc.resolve(oldNode.pos).depth,
               };
 
+              // Hacky fix to avoid processing certain transactions created by ordered list indexing plugin.
```

Comment:
Can't we re include the above then?

```
if (transaction.getMeta("orderedListIndexing") !== undefined) {
          return prev;
        }
```

Reply:
If we need to keep it, can you comment (explain what they do) indexInitialized / depthChanged / update?

Reply:
```
if (transaction.getMeta("orderedListIndexing") !== undefined) {
    return prev;
}
```

This prevents all transactions from the `OrderedListItemIndexPlugin`, but we actually want to process some of them for the animations to work, which are let through by `indexInitialized` and `depthChanged`.